environment/prod/deployment/prod/Makefile

```diff
@@ -16,8 +16,8 @@ DOCKER_EXEC_SCRIPT = docker compose --file ${DOCKER_COMPOSE_FILENAME} exec --wor
 CURRENT_DIR = $(shell pwd)
 
 prod-deploy: decrypt-secrets create-deployment-file
-	@docker compose --file ${DOCKER_COMPOSE_FILENAME} pull && \
-	docker compose --file ${DOCKER_COMPOSE_FILENAME} up --detach && \
+	@docker compose -p keating-prod --file ${DOCKER_COMPOSE_FILENAME} pull && \
+	docker compose -p keating-prod --file ${DOCKER_COMPOSE_FILENAME} up --detach && \
```

Comment:
It will be easier to understand if you will use fill option name: `--project-name`.
Also it would be good to move this project name to some variable.